src/class_resolver/api.py

```diff
@@ -358,6 +358,25 @@ def make_many(
             for _result_tracker, _result_tracker_kwargs in zip(_query_list, _kwargs_list)
         ]
 
+    def make_rst_table(self, **kwargs) -> str:
+        """
+        Render the table of options in a format suitable for Sphinx documentation.
+
+        :param kwargs:
+            additional keyword-based parameters passed to :func:`tabulate.tabulate`.
+        """
+        import tabulate
+
+        return tabulate.tabulate(
+            (
+                (norm_key, f":class:`~{cls.__module__}.{cls.__qualname__}`",)
+                for norm_key, cls in self.lookup_dict()
+            ),
+            headers=["key", "class"],
+            tablefmt="rst",
```

Comment:
rename this function and make the tablefmt an argument that can be chosen, maybe this is also nice for generating markdown too, e.g., in pykeen's readme

Reply:
done in https://github.com/cthoyt/class-resolver/pull/66/commits/70f54a7a6a6ecd21681b809a1b569ec9cb61a100